hwbench/environment/tests/vendors/Dell/C6420/oem_system

```diff
@@ -0,0 +1,907 @@
+{
```

Comment:
How do you generate the test files to be in python format and not json ?

Reply:
that's a curl output, slightly modified:

- null -> None
- false -> False
- true -> True

Reply:
This makes me wonder if we shouldn't change this to use json directly in the future. Do you remember why we picked python ast over json to load test files?